coremltools/converters/mil/frontend/torch/test/test_torch_ops.py

```diff
@@ -396,6 +402,30 @@ def test_mv(self, matrix_shape, backend):
         TorchBaseTest.run_compare_torch((matrix, vector), model, backend=backend, input_as_shape=False)
 
 
+class TestCossim(TorchBaseTest):
+    
+    @pytest.mark.parametrize("backend", backends)
+    def test_cosine_similarity(self, backend):
+        class CosineSimilarity(nn.Module):
+            def __init__(self):
+                super(CosineSimilarity, self).__init__()
+                # self.flatten = nn.Flatten()
+                self.cossim = torch.nn.CosineSimilarity()
+
+            def forward(self, x,y):
+                out = self.cossim(x,y)
+                return out
+        model = CosineSimilarity()
+        input1 = generate_input_data((2,3))
+        input2 = generate_input_data((2,3))
+        model_spec= TorchBaseTest.run_compare_torch(
+            [input1,input2], 
```

Comment:
This is a very minor comment, but per [PEP8](https://pep8.org) (the common Python style guide), you want a space after the first comma on this line, i.e `[input1, input2],`

Reply:
done